scheduler/service/service.go

```diff
@@ -379,112 +380,116 @@ func (s *Service) LeaveTask(ctx context.Context, req *rpcscheduler.PeerTarget) e
 	return nil
 }
 
-// StatPeerTask checks if the given task exists in P2P network
+// StatPeerTask checks the current state of the task
 func (s *Service) StatPeerTask(ctx context.Context, req *rpcscheduler.StatPeerTaskRequest) (*base.GrpcDfResult, error) {
-	log := logger.With("function", "StatPeerTask", "TaskID", req.TaskId)
-
-	// Check if task exists
 	task, loaded := s.resource.TaskManager().Load(req.TaskId)
-	if !loaded || task == nil {
-		msg := "task not found in P2P network"
-		log.Info(msg)
+	if !loaded {
+		msg := fmt.Sprintf("task %s not found", req.TaskId)
+		logger.Info(msg)
+		// TODO(eryugey) use dferrors.Newf
 		return common.NewGrpcDfResult(base.Code_PeerTaskNotFound, msg), nil
 	}
+
 	if task.FSM.Current() != resource.TaskStateSucceeded {
-		msg := fmt.Sprintf("task found but not in %s state: %s", resource.TaskStateSucceeded, task.FSM.Current())
-		log.Info(msg)
+		msg := fmt.Sprintf("task has been found but state is %s", task.FSM.Current())
```

Comment:
I'd like to see the expected state in log msg, e.g. TaskStateSucceeded, so we know what's expected and what we actually have.

Also I'd like print taskID as well, so we know which task is in unexpected state, it makes debug easier.